### DISCUSSION TITLE
[Share] Create New Standalone Note

### DISCUSSION THREAD

**Discussion by ulahcherubim** [windingwind/zotero-actions-tags#274] [Action Scripts]
### Description

Creates New Standalone Note

### Event

None

### Operation

Script

### Data

```javascript
/**
 * Creates Unfiled New Standalone Note .
 * @author windingwind, ulahcherubim
 * @usage 
 * @link https://github.com/windingwind/zotero-actions-tags/discussions/274
 * @see https://github.com/windingwind/zotero-actions-tags/discussions/271
 */
const ZoteroPane = require("ZoteroPane");
ZoteroPane.newNote(false, null, "", "");
```

### Anything else

This action can be developed further using information available there: https://github.com/zotero/zotero/blob/a7d59a90d37695e4eaa0e42d9e2301b049d0a0e7/chrome/content/zotero/zoteroPane.js#L4286

**Comment by muyi123567** [comment]
Thanks for your work, I found the feature I was looking for!
